manual/src/threelang/tfgrid/gateway_fqdn.md

```diff
@@ -0,0 +1,31 @@
+# Gateway FQDN Namespace
+
+- To deploy a gateway fqdn workload, use the Gateway FQDN namespace
+
+## Create Operation
+
+- action name: !!tfgrid.gateway_fqdn.create
```

Comment:
Maybe rename to tfgrid.gateway.fqnd.create? And the other to tfgrid.gateway.name.create.

Reply:
i believe this was suggested before, but i think we reached a conclusion that for the sake of simplicity we should put a constraint of two or three words per action name, no?

Reply:
Ok for sake of simplicity let's use the tfgrid.gateway_fqdn.create and tfgrid.gateway_name.create